mk/shared/glestkeys.ini

```diff
@@ -26,20 +26,30 @@ GroupUnitsKey7='6'
 GroupUnitsKey8='7'
 GroupUnitsKey9='8'
 GroupUnitsKey10='9'
-CameraRotateLeft=A
-CameraRotateRight=D
+CameraRotateLeft=O
+CameraRotateRight=P
 CameraRotateUp=S
 CameraRotateDown=W
+HotKeyBuild1=Q
+HotKeyBuild2=W
+HotKeyBuild3=E
+HotKeyBuild4=R
+HotKeyBuild5=A
+HotKeyBuild6=S
+HotKeyBuild7=D
+HotKeyBuild8=F
+HotKeyBuild9=Z
+HotKeyBuild10=X
 HotKeyCenterCameraOnSelection=G
 HotKeySelectIdleHarvesterUnit=I
 HotKeySelectBuiltBuilding=B
 HotKeyShowDebug=?
 HotKeyDumpWorldToLog=\
 HotKeyRotateUnitDuringPlacement=R
-HotKeySelectDamagedUnit=D
+HotKeySelectDamagedUnit=U
 HotKeySelectStoreUnit=T
-HotKeySelectedUnitsAttack=A
-HotKeySelectedUnitsStop=S
+HotKeySelectedUnitsAttack=I
```

Comment:
Good catch, the keys that I remapped were only temporary to not have conflicts with the grid keys

Reply:
For this key in particular I didn't know where to move it, since it's replaced by the equivalent grid key

Reply:
Afraid I spotted a few other double mapped keys: P, W, and S.
See comment about grid mapping for "I" solution.

Reply:
The camera hotkeys aren't that much of a problem since they only conflict in free camera mode. What do you think of leaving those as they were? There's not a lot of keys left. The drawback is that you don't have the command hotkeys in free camera mode but I think that's reasonable.

Reply:
@Jammyjamjamman I'm waiting on your feedback to change this.

Reply:
Yeah I like that idea. Actually I'd prefer the command hotkeys to be overridden than pause.

Reply:
Alright, fixed in  https://github.com/MegaGlest/megaglest-source/pull/216/commits/33dfb0e264c905f75bcc0a7245dea7e875d66e32

With this I think I'm pretty much done with this feature. If you have any suggestion let me know.